local-source-proxy/install.hbs.md

```diff
@@ -105,11 +105,11 @@ read access.
 In such cases, `pull_secret` can be specified, and `source-controller` uses the `pull_secret`
 to pull source artifacts for deployment.
 
-`pull_secret` uses Docker registry credentials secret referenced by `name` and `namespace`.
-Set `pull_secret.create_export` as `true` if a `SecretExport` resource needs to be created in its
-namespace to allow the secret to be exported to developer namespaces by using Namespace Provisioner
-or other means.
-<!-- Who is exporting the secret? People or machines or both? I need to know to make the sentence easier to read. -->
```

Comment:
This action is completed by the TAP installer if `create_export` is set to true. Otherwise, operator needs to manually create the applicable `SecretExport` resource in the referenced `namespace`. This `SecretExport` resource is necessary for the TAP installer to setup the configuration values required by Local Source Proxy.